src/components/AppFrame/AppFrame.js

```diff
@@ -57,10 +59,12 @@ const AppFrame = ({
 	scopeFilterPlaceholder,
 	noScope,
 }) => {
+	const locale = useSelector(currentLocale);
 	const applications = unwrapImmutable(useSelector(localizedAppSelector));
 	const [helpUrl] = useApplicationHelpUrl(applicationId);
 	useLoader(getApplications(), state => localizedAppSelector(state).size);
 	const [open, toggle, reset] = useToggle(initOpen);
+	useLoader(getVersionInfo(locale), () => helpUrl !== null);
```

Comment:
If the language is changed, this will not get version info for the new language: /fr